stix_shifter/stix_translation/stix_translation.py

```diff
@@ -16,6 +16,8 @@
 DIALECTS = 'dialects'
 SUPPORTED_ATTRIBUTES = "supported_attributes"
 MAPPING_ERROR = "Unable to map the following STIX objects and properties:"
```

Comment:
I think you need to now add "and operators" to this error message.

Reply:
I actually have added that later where exception thrown in line 110-
```
raise DataMappingException(
                                 "{} {} and Operators: {} to data source fields".format(MAPPING_ERROR, unmapped_stix_collection, unmapped_operator_collection)
                             )
```